test/integration/apiserver/apply/apply_test.go

```diff
@@ -284,6 +284,37 @@ func TestCreateOnApplyFailsWithUID(t *testing.T) {
 	}
 }
 
+func TestCreateOnUpdateFails(t *testing.T) {
```

Comment:
That clearly seems like the wrong place to do that. I'm hoping we already have tests for that, and I'm pretty sure they have nothing to do with Apply.

Reply:
Oh, that slipped inside. I just wanted to be sure it behaves like that. I'd hope there is a test already, yes.

Reply:
do we have a test that verifies create-on-apply works with a custom resource?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apply/apply_crd_test.go#L72-L88

Reply:
great, looks like all the scenarios in that file test that, which is good... I'm unsure all the field manager paths are exercised when the schema is absent or non-structural - https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apply/apply_crd_test.go#L240-L248